demo/d2l-scroll-wrapper.html

```diff
@@ -3,10 +3,11 @@
 	<head>
 
 		<script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+		<script src="update-direction.js"></script>
 		<link rel="import" href="../d2l-scroll-wrapper.html">
-		<link rel="import" href="../../d2l-demo-template/d2l-demo-template.html">
+		<link rel="import" href="../../d2l-typography/d2l-typography.html">
 		<custom-style>
-		<style is="custom-style">
+		<style is="custom-style" include="d2l-typography">
```

Comment:
You may want to set a `font-size: 20px` on the body as well:
https://github.com/Brightspace/generator-polymer-init-d2l-hybrid-element/blob/master/generators/app/templates/demo/index.html#L18